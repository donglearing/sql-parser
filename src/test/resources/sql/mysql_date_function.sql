select a from T t, F f where date(t.b) = f.c;