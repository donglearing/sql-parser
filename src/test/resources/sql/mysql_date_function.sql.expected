SELECT a FROM T AS t, F AS f WHERE date(t.b) = f.c